core/frontend/src/views/ExtensionManagerView.vue

```diff
@@ -0,0 +1,212 @@
+          class="pa-2"
+        >
+          <extension-card
+            :extension="extension"
+            @clicked="showModal(extension)"
+          />
+        </v-col>
+      </v-row>
+      <v-container
+        v-if="manifest.length === 0"
+        class="text-center"
+      >
+        <p class="text-h6">
+          No Extensions available.
+        </p>
+      </v-container>
+    </div>
+    <v-row>
+      <v-col
+        v-if="tab === 0"
+        class="pa-5"
+      >
+        <v-row
+          v-for="extension in installed_extensions"
+          :key="extension.name"
+          dense
+        >
+          <v-col
+            class="pa-2"
+          >
+            <v-card>
+              <v-card-title>
+                {{ extension.name.replace('/', '') }}: <span style="color: grey;"> {{ extension.tag }}</span>
+              </v-card-title>
+              <v-card-text>
+                Permissions:
+                <pre> {{ extension.permissions }} </pre>
+              </v-card-text>
+              <v-card-actions>
+                <v-btn @click="uninstall(extension)">
+                  Uninstall
+                </v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-col>
+        </v-row>
+        <v-container
+          v-if="installed_extensions.length === 0"
+          class="text-center"
+        >
+          <p
+            v-if="dockers_fetch_done"
+            class="text-h6"
+          >
+            No Extensions installed.
+          </p>
+          <p
+            v-else
+            class="text-h6"
+          >
+            Fetching Extensions
+          </p>
+        </v-container>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+import axios from 'axios'
+import Vue from 'vue'
+
+import ExtensionCard from '@/components/kraken/ExtensionCard.vue'
+import ExtensionModal from '@/components/kraken/ExtensionModal.vue'
+import Notifier from '@/libs/notifier'
+import { kraken_service } from '@/types/frontend_services'
+import back_axios from '@/utils/api'
+
+import { ExtensionData } from '../types/kraken'
+
+const API_URL = '/kraken/v1.0'
+
+const notifier = new Notifier(kraken_service)
+
+export default Vue.extend({
+  name: 'ExtensionManagerView',
+  components: {
+    ExtensionCard,
+    ExtensionModal,
+  },
+  data() {
+    return {
+      tab: 0,
+      show_dialog: false,
+      installed_extensions: [] as ExtensionData[],
+      selected_extension: null as (null | ExtensionData),
+      manifest: [] as ExtensionData[],
+      dockers_fetch_done: false,
+    }
+  },
+  mounted() {
+    this.fetchManifest()
+    this.fetchInstalledExtensions()
+  },
+  methods: {
+    async fetchManifest(): Promise<void> {
+      await back_axios({
+        method: 'get',
+        url: `${API_URL}/extensions_manifest`,
+        timeout: 3000,
+      })
+        .then((response) => {
+          const manifest = response.data
+          this.manifest = manifest
```

Comment:
Is there a need for the intermediary variable ?

Reply:
nope, consider it gone.